tests/e2e/distribution/helpers.go

```diff
@@ -0,0 +1,32 @@
+package distribution
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	distrcli "github.com/cosmos/cosmos-sdk/x/distribution/client/cli"
+)
+
+func MsgWithdrawDelegatorRewardExec(clientCtx client.Context, valAddr fmt.Stringer, extraArgs ...string) ([]byte, error) {
```

Comment:
Why do we need to duplicate that?

Reply:
we can, I accidentally made this PR r4R. I was about to do these changes.